queue-worker/main.go

```diff
@@ -90,11 +90,15 @@ func main() {
 		req := queue.Request{}
 		json.Unmarshal(msg.Data, &req)
 		fmt.Printf("Request for %s.\n", req.Function)
-		urlFunction := fmt.Sprintf("http://%s%s:8080/", req.Function, functionSuffix)
+		functionURL := fmt.Sprintf("http://%s%s:8080/?%s", req.Function, functionSuffix, req.QueryString)
 
-		request, err := http.NewRequest("POST", urlFunction, bytes.NewReader(req.Body))
+		request, err := http.NewRequest("POST", functionURL, bytes.NewReader(req.Body))
```

Comment:
Can you also change "POST" to http.MethodPost (or whatever it's actually called)?

Reply:
it seems @johnmccabe has beat me to it with https://github.com/openfaas/nats-queue-worker/pull/8. I will clean up the other two instances though